src/rc_allocator.c

```diff
@@ -240,13 +240,16 @@ rc_allocator_init_meta_page(rc_allocator *al)
    platform_assert((1 + RC_ALLOCATOR_MAX_ROOT_IDS) * al->cfg->io_cfg->page_size
                    <= al->cfg->io_cfg->extent_size);
 
-   al->meta_page = TYPED_ALIGNED_ZALLOC(al->heap_id,
-                                        al->cfg->io_cfg->page_size,
-                                        al->meta_page,
-                                        al->cfg->io_cfg->page_size);
+   platform_memfrag memfrag_meta_page;
+   al->meta_page = TYPED_ALIGNED_ZALLOC_MF(al->heap_id,
+                                           al->cfg->io_cfg->page_size,
+                                           al->meta_page,
+                                           al->cfg->io_cfg->page_size,
+                                           &memfrag_meta_page);
```

Comment:
Here's an example of an allocation interface where the macro cannot simply synthesize the `platform_memfrag* mf` name using `##v` notation.

The caller has to supply the required `memfrag *mf` arg by name; i.e. `&memfrag_meta_page`.